src/styles/variables.scss

```diff
@@ -0,0 +1,3 @@
+$mainColor: #060b35;
+$otherColor: #616070;
```

Comment:
let's come up with a better name for this color

Reply:
I did